### DISCUSSION TITLE
strap http(s) from {{menuUrl}} helper ?

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1382] [Help]
Hi, can I force the {{menuUrl}} helper to show an URL without http(s) ?
As far as I understand it, the output is https//:site.tld/page. I would prefer to have the link without the web protocol, esp for the front-page and internal links. for external links the https shall stay in the URL

**Comment by dziudek** [comment]
Well, the easiest way would be to use the Smart Replacer plugin: https://marketplace.getpublii.com/plugins/smart-replacer/ and replace all occurences of `href="https://example.com` to `href="example.com` - it will replace it on the menu and in the website content.

Other solutions most probably will require writing custom helpers and won't be global

**Reply by 3muchfun** [reply]
Thanks for the quick answer, but I am not sure whether I am willing to pay over 70€.
Maybe somebody else will find a free solution ...

**Reply by bobmitro** [reply]
€59 only :)  I know it costs a bit, but it is a powerful tool, e.g., it would help you add an icon to the external links you asked about in another [thread](https://github.com/GetPublii/Publii/discussions/1380). I know you'll say I'm advertising a premium product; yes, I'm advertising it because its possibilities are incredible.

**Reply by 3muchfun** [reply]
€59 + VAT = 70.x€
Anyway, I found a free solution for my problem, see #1380

**Reply by bobmitro** [reply]
I came across this solution because I was the one who provided it to you. :)
